submissions/madmaxWMFU/google-popup/css/style.css

```diff
@@ -98,6 +95,15 @@
     display: none;
 }
 
+
+/* .otherApps:target {
```

Comment:
Avoid committing commented code. Use another extra branch for this.

Reply:
This mistake was resolved. If you can see the last changes afterward, everything is fine there